README.md

```diff
@@ -1,7 +1,50 @@
 # sknnr-spatial
 
-[![Build status](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml/badge.svg)](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml)
+[![Build status](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml/badge.svg)](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml) [![Documentation status](https://readthedocs.org/projects/sknnr-spatial/badge/?version=latest)](https://sknnr-spatial.readthedocs.io/)
 
-Spatial predictions using [scikit-learn kNN regression](https://github.com/lemma-osu/sknnr).
+> ⚠️ **WARNING: sknnr-spatial is in active development!** ⚠️
 
-> ⚠️ **WARNING: sknnr-spatial is in active development!** ⚠️
\ No newline at end of file
+## Features
+
+- 🗺️ Raster predictions from [scikit-learn](https://scikit-learn.org/stable/) estimators 
+- ⚡ Parallelized functions + larger-than-memory data using [Dask](https://www.dask.org/)
+- 🌐 Automatic handling of projections, band names, and masks
+
+## Quick-Start
+
+1. Wrap a `scikit-learn` estimator prior to fitting to enable raster-based predictions:
+
+    ```python
+    from sklearn.ensemble import RandomForestRegressor
+    from sknnr_spatial import wrap
+
+    est = wrap(RandomForestRegressor())
+    ```
+
+2. Load a [custom dataset](https://sknnr-spatial.readthedocs.io/en/latest/api/datasets/swo_ecoplot) of features and targets and fit the wrapped estimator:
+
+    ```python
+    from sknnr_spatial.datasets import load_swo_ecoplot
+
+    X_image, X, y = load_swo_ecoplot(as_dataset=True)
+    est.fit(X, y)
+    ```
+
+3. Generate predictions from a `numpy` or `xarray` raster with predictors as bands:
+
+    ```python
+    pred = est.predict(X_image)
+    pred["PSME_COV"].plot()
```

Comment:
I think the user needs `matplotlib` installed for this to work, right?  Where does it make sense to have `matplotlib` specified as an optional dependency?  Or do we assume that the user will have `matplotlib` installed as sidecar?

Reply:
Great point. I was planning to add a `tutorial` group to the optional dependencies (which would include `matplotlib` and `sknnr-spatial[datasets]`) when we start adding notebooks, but of course you're right that we should make sure the quickstart is actually runnable!

I can add those optional dependencies now along with a step to install them.

Reply:
Just to be clear, the line:
```
pip install "sknnr-spatial[tutorials] @ git+https://github.com/lemma-osu/sknnr-spatial.git"
```
won't yet work?  I'm getting:
```
WARNING: sknnr-spatial 0.1.0+dev does not provide the extra 'tutorials'
```
when trying to install into a fresh new virtualenv

Reply:
Yeah, that makes sense. You should be able to test it temporarily by specifying the `readme` branch:

```bash
pip install "sknnr-spatial[tutorials] @ git+https://github.com/lemma-osu/sknnr-spatial.git@readme"
```

Once it's merged into `main`, that won't be needed and it should work as is :)

Reply:
Ah, that makes sense.  My brain is working at half-capacity after the holidays ...